### ISSUE TITLE
NVJPEG error "4" : NVJPEG_STATUS_JPEG_NOT_SUPPORTED 

### ISSUE THREAD

**Issue by amnash** [NVIDIA/DALI#1204]
Hi there,
I'm using DALI to train a resnet50 on imagenet, using mxnet 1.3.0 and cuda10 on a V100 GPU. I encounter NVJPEG error "4" on various images. The error code descriptor points to JPEG not being supported. Any idea what exactly the incompatiblility might be?

Many thanks

**Comment by awolant** [comment]
Hi, thanks for the question.
I remember we had some problems with ability of nvJPEG to decode some of the progressive jpeg images. Maybe that is it?
It would be helpful if you could show exact images?

**Comment by amnash** [comment]
Thank you so much for your response!

I looked into the matter and it turned out that I had a few faulty images in the dataset I was using (which I used to create the .rec files). The training runs soomthly now after replacing those.

I basically turned off random shuffle, used a minibatch size of 1, cross-referenced with the .lst file to find the faulty image. I was wondering if there is an easy way to do this?
